Sources/Tracking/CustomerIO.swift

```diff
@@ -22,7 +24,18 @@ public protocol CustomerIOInstance: AutoMockable {
         jsonEncoder: JSONEncoder?,
         onComplete: @escaping (Result<Void, CustomerIOError>) -> Void
     )
-    func clearIdentify()
+
+    // sourcery:Name=screenView
```

Comment:
Why not use `screen` as the name of the mocked function to match the name of the function here?

Reply:
haha whoops, forgot to update the sourcery part when I changed